src/src/single-spa-angular.ts

```diff
@@ -42,6 +43,10 @@ export function singleSpaAngular<T>(userOptions: SingleSpaAngularOptions<T>): Li
     throw Error(`single-spa-angular must be passed the NgZone option`);
   }
 
+  if (options.Router && !options.NavigationStart) {
```

Comment:
This makes this change a breaking change - was that your intent? If we don't plan on this being a semver major, we should not throw this error, but rather only use NavigationStart when it's provided.